packages/react-native-sdk/src/components/Livestream/HostLivestream/HostLivestream.tsx

```diff
@@ -72,7 +71,7 @@ export const HostLivestream = ({
   };
 
   return (
-    <SafeAreaView
```

Comment:
@khushal87 we should never use safeareaview inside the SDK

we have to let the user do the safe area stuff.. as these components can be inside bottom tabs.. half height of screens etc